pom.xml

```diff
@@ -142,5 +148,10 @@
       <version>${junitVersion}</version>
       <scope>test</scope>
     </dependency>
-  </dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.6.1</version>
+    </dependency>
```

Comment:
Why were the logging dependencies added?

Reply:
Are those meant for testing scope?

Reply:
see above answer

Reply:
yes; I think only for testing scope. can we limit the dependency to that scope?